configure.ac

```diff
@@ -214,7 +202,6 @@ AC_CONFIG_FILES([Makefile
 	src/daemon/Makefile
 	src/tools/Makefile
 	src/pam/Makefile
-	src/bindings/Makefile
```

Comment:
`src/bindings/` directory also needs to be removed, because the new binding code is placed under `src/python`

Reply:
I'm not seeing `src/bindings/` in anything committed into github (related to this patchset).  Is it just an artifact laying around in your sandbox from an old build?

Reply:
Ah !! sorry for the noise, it was from the old build.